EnglishTranslation-master/SKILL.tsv

```diff
@@ -2300,7 +2300,7 @@ SKILL_20150317_002299	Strength of enemies within [Ogouveve] decreases by 2 per a
 SKILL_20150317_002300	Damballa: Chance of Zombification	댐발라: 좀비 출현 확률				
 SKILL_20150317_002301	Chance of new zombie appearing increase by 2% per attribute level when using [Damballa].	스킬 [댐발라]의 새 좀비 출현 확률이 특성 레벨 당 2% 증가합니다.				
 SKILL_20150317_002302	Bwa Kayiman: zombie defense	브와 카이망: 좀비 방어				
-SKILL_20150317_002303	Physical defense of zombies in [Bwa Kayiman] increases by 3 per Attribute level.	스킬 [브와 카이망]에 사용되는 좀비들의 물리 방어력이 특성 레벨 당 3 증가합니다.				
+SKILL_20150317_002303	Physical Defense of zombies in [Bwa Kayiman] increases by 3 per Attribute level.	스킬 [브와 카이망]에 사용되는 좀비들의 물리 방어력이 특성 레벨 당 3 증가합니다.				
```

Comment:
`Increases the Physical Defense of Zombies from [Bwa Kayiman] by 3 per attribute level.`